.github/workflows/lint.yml

```diff
@@ -0,0 +1,16 @@
+name: ❄️ Lint
+
```

Comment:
Please have a proper filename, such as markdown-linter?

Reply:
When I add another linter it will go in this same file. 
 So it is best just to call it `Lint` or `Linter`
There are `jobs` with sub headings where `Markdown` goes.

Reply:
Oh I got it.